package.json

```
@@ -228,6 +228,7 @@
     "**/@definitelytyped/header-parser": "^0.0.163",
     "**/@definitelytyped/typescript-versions": "^0.0.163",
     "**/@definitelytyped/utils": "^0.0.163",
+    "**/@types/node": "<19",
```

You can immediately specify the actual version here because with the next v18 bump, renovate will anyways change it to a specific version: https://github.com/mui/mui-x/pull/9536/files
```suggestion
    "**/@types/node": "^18.16.19",
```